doc/architecture/gems/dry_crud.md

```diff
@@ -0,0 +1,42 @@
+# DRY CRUD
+
+[Homepage](https://github.com/codez/dry_crud)
+
+DRY_CRUD is a gem which can be used to generate a CRUD Controller which enables the CRUD actions for
+our rails models. After this controller has been generated we can remove the gem and make full use of our
+new controller.
```

Comment:
Da würde ich eher aus der Perspektive von hitobito schreiben. Also z.B. 

dry_crud was used to generate several classes like the `CrudController` to generalize CRUD actions for our models.